R/seeker.R

```diff
@@ -506,6 +506,16 @@ seeker = function(params, parentDir = '.', dryRun = FALSE) {
 
   ####################
   fwrite(metadata, metadataPath)
+  commandVersions = checkDefaultCommands()
```

Comment:
This stuff should go in its own function. I also think commandVersions is not the most fitting name.